libpod/healthcheck.go

```diff
@@ -107,14 +107,20 @@ func (c *Container) runHealthCheck() (HealthCheckStatus, error) {
 		capture       bytes.Buffer
 		inStartPeriod bool
 	)
-	hcStatus, err := checkHealthCheckCanBeRun(c)
-	if err != nil {
-		return hcStatus, err
-	}
 	hcCommand := c.HealthCheckConfig().Test
-	if len(hcCommand) > 0 && hcCommand[0] == "CMD-SHELL" {
-		newCommand = []string{"sh", "-c", strings.Join(hcCommand[1:], " ")}
-	} else {
+	if len(hcCommand) < 1 {
+		return HealthCheckInternalError, errors.Errorf("container %s has a corrupted healthcheck command", c.ID())
+	}
+	switch hcCommand[0] {
+	case "NONE":
```

Comment:
Should we alias the empty string to this? It's definitely not properly-formatted, but printing a good error seems better than trying to execute an empty command

Reply:
It now uses the same error message.

Reply:
@mheon I think I finally got what you meant. Fixing...